superset-frontend/package.json

```diff
@@ -121,6 +121,7 @@
     "mousetrap": "^1.6.1",
     "mustache": "^2.2.1",
     "omnibar": "^2.1.1",
+    "path": "^0.12.7",
```

Comment:
why do we need to add this to the deps? I don't see it used anywhere?

Reply:
It is used in `webpack.config.js` and it triggers `import/no-extraneous-dependencies` rule if it's not listed in deps

Reply:
@kgabryje Since most production servers probably use a static build rather than the webpack server, I wonder if it makes sense to move these to devDependencies. Does that still cause an eslint error?

If it does still cause an error, I'd be tempted to leave it as a devDependency anyway and/or use the ol' `// eslint-disable-next-line import/no-extraneous-dependencies` to quiet it.